test/src/Command/QueueCleanCommandTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace tests\eLife\Bus\Command;
+
+use eLife\Bus\Command\QueueCleanCommand;
+use eLife\Bus\Queue\InternalSqsMessage;
+use eLife\Bus\Queue\Mock\WatchableQueueMock;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @covers \eLife\Bus\Command\QueueCleanCommand
+ */
+class QueueCleanCommandTest extends TestCase
+{
+    /** @var Application */
+    private $application;
+    /** @var QueueCleanCommand */
+    private $command;
+    /** @var CommandTester */
+    private $commandTester;
+    /** @var WatchableQueueMock */
```

Comment:
we will rename this class to `InMemoryQueue` since it's a fake implementation, not a mock

Reply:
That sounds like a good refactor.

Reply:
I might try and take that on in a future PR to extend the queue commands

Reply:
no hurry, would have to check how it's used (if it is) in dependent projects